RealestateApi/Model/RealEstate.cs

```diff
@@ -10,7 +10,7 @@ public class RealEstate
         public DateTime Build_Date { get; set; }
         public decimal Price { get; set; }     
         public int SquareMeter { get; set; }
-        public string EnergyClass { get; set; } = null!;
+        public string EnergyClass { get; set; }
```

Comment:
Why did you delete "= null" ?

Reply:
A mistake made with the last merge I did, and didn't noticed.